src/api/app/views/webui/request/beta_show_tabs/_conversation_aside.html.haml

```diff
@@ -3,16 +3,14 @@
   - if request_reviews.present?
     .mt-4
       .text-end
-        = render AddReviewDropdownComponent.new(bs_request: bs_request, user: User.session,
-                                              can_add_reviews: can_add_reviews,
-                                              my_open_reviews: my_open_reviews)
+        = render AddReviewDropdownComponent.new(bs_request: bs_request, user: User.session, my_open_reviews: my_open_reviews)
       = render AddReviewCollapsibleComponent.new
       .mb-2
         %h6
           Reviewers
       .mb-4
         = render partial: 'webui/request/beta_show_tabs/review_summary', collection: request_reviews, as: :review
-  = render partial: 'webui/request/beta_show_tabs/ask_for_review', locals: { can_add_reviews: can_add_reviews }
+  = render partial: 'webui/request/beta_show_tabs/ask_for_review', locals: { can_add_reviews: policy(bs_request).add_reviews? }
```

Comment:
What about wrapping the `render partial` with `if policy(...` and get rid of the `locals` completely here and inside the `ask_for_review` partial? If it was a view component that can compute its own policy it would be better (like the `AddReviewDropdownComponent`), but this is a simple stateless partial I'd rather keep it _ignorant_ about the condition.

Reply:
done